test/parallel/test-argv0-property.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.ok(process.hasOwnProperty('argv0'));
+assert.throws(() => {
+  process.argv0 = 'anything';
+}, /read only/);
```

Comment:
If `'#<process>'` is stable then the full string `"TypeError: Cannot assign to read only property 'argv0' of object '#<process>'"` would be best.